docs/integrations/destinations/s3-migrations.md

```diff
@@ -0,0 +1,203 @@
+
+Formerly, unions in parquet were represented as anonymous disjoint records. For example:
+
+| Airbyte Union Type                                                                                                                     | Old Parquet Type                                                                                                         |
+|----------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| `{"oneOf": [ {"type": "integer"}, {"type": "string"} ] }`                                                                              | `Optional Record { member0: Optional Int32, member1: Optional String }`                                                  |
+| `{"oneOf": [ {"type": "boolean"}, {"type": "object", "properties": { "id": { "type": "integer" }, "name": { "type": string" } } } ] }` | `Optional Record { member0: Optional Boolean, member1: Optional Record { id: Optional Uint32, name: Optional String } }` |
+
+Now unions will be represented as typed disjoint records with named fields. For example:
+
+| Airbyte Union Type                                                                                                                     | New Parquet Type                                                                                                                               |
+|----------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| `{"oneOf": [ {"type": "integer"}, {"type": "string"} ] }`                                                                              | `Optional Record { type: Optional String, integer: Optional Int32, string: Optional String }`                                                  |
+| `{"oneOf": [ {"type": "boolean"}, {"type": "object", "properties": { "id": { "type": "integer" }, "name": { "type": string" } } } ] }` | `Optional Record { type: Optional String, boolean: Optional Boolean, object: Optional Record { id: Optional Uint32, name: Optional String } }` |
+
+Where
+* only one field will always be set, and
+* the value of the `type` field will always be equal the name of the set field
+
+Following the examples above:
+
+| Input Data                    | New Output Data                                                 |
+|-------------------------------|-----------------------------------------------------------------|
+| `1`                           | `{ "type": "integer", "integer": 1 }`                           |
+| `{"id": 10, "name": "Alice"}` | `{ "type": "object", "object": { "id": 10, "name": "Alice" } }` |
+
+#### Disjoint record type names
```

Comment:
```suggestion
#### Disjoint Record-Type Names
```

Reply:
Capitalization yes. It's "type names" for records, not the name of record types, so I'm not sure the hyphen makes sense.